content/webapp/components/IIIFItem/IIIFItem.tsx

```diff
@@ -57,19 +65,62 @@ const Choice: FunctionComponent<
   return null;
 };
 
+const IIIFImage: FunctionComponent<{
+  index: number;
+  item: ItemProps['item'];
+  canvas: TransformedCanvas;
+  setImageRect?: (v: DOMRect) => void;
+  setImageContainerRect?: (v: DOMRect) => void;
+}> = ({ index, item, canvas, setImageRect, setImageContainerRect }) => {
+  const [ocrText, setOcrText] = useState(missingAltTextMessage);
+  const imageService = getImageServiceFromItem(item);
+  const imageUrl = imageService?.['@id'] || '';
+  const infoUrl = convertRequestUriToInfoUri(imageUrl);
+  const urlTemplate = imageUrl ? iiifImageTemplate(imageUrl) : undefined;
+  useEffect(() => {
+    const fetchOcr = async () => {
+      const ocrText = await fetchCanvasOcr(canvas);
+      const ocrString = transformCanvasOcr(ocrText);
+      setOcrText(ocrString || missingAltTextMessage);
+    };
+    fetchOcr();
+  }, []);
+
+  if (urlTemplate) {
+    return (
+      <ImageViewer
+        infoUrl={infoUrl}
+        id={imageUrl}
+        width={canvas.width || 0}
+        height={canvas.height || 0}
+        index={index}
+        alt={ocrText}
+        urlTemplate={urlTemplate}
+        setImageRect={setImageRect}
+        setImageContainerRect={setImageContainerRect}
+      />
+    );
+  } else {
+    return <img src={item.id} alt={ocrText} />;
+  }
+};
+
+// Some of our ContentResources can have a type of 'Audio':
+// https://iiif.wellcomecollection.org/presentation/v3/b17276342
```

Comment:
What is this in reference too, why are they to be singled out?

Reply:
'Audio' isn't officially a valid type, so typescript was complaining - but it is something we have, so I added it

Reply:
Ahhh I see! Could we add that in the comment? 🙏